testUtils/setupTests.js

```diff
@@ -16,6 +16,8 @@ Object.defineProperty(window.navigator, 'onLine', {
   },
 });
 
+window.scrollTo = jest.fn();
```

Comment:
This method does not exist in `jsdom` and was making the `Select` input throw errors in tests (it uses it for scrolling the dropdown into view).